engine-src/Game/LambdaHack/Client/UI/Frontend/Sdl.hs

```diff
@@ -198,6 +200,7 @@ startupFun coscreen soptions@ClientOptions{..} rfMVar = do
         }
   swindow <- SDL.createWindow title windowConfig
   srenderer <- SDL.createRenderer swindow (-1) rendererConfig
+  when sfullscreen (SDL.rendererLogicalSize srenderer SDL.$= Just screenV2)
```

Comment:
Why is this needed? Given that `srenderer` was created based on a window created with `screenV2` size, I'd expect it to be already set, but I generally don't understand this setting and its function, so I may be missing something. OTOH, if we change the font scale in-game I fully expect we'd need to call `rendererLogicalSize`.

Reply:
This is needed so that the game "stretches" to take up the full screen
space (if font scale is set improperly, you are on retina screen, etc.). This also centers the game
vertically (so it isn't in the top left corner) and adjusts all the maths internally so we don't have to add offsets & scaling math to our UI code.

Reply:
Oh, neat. Let's keep it.